docs/content/guides/3.performance/2.core-web-vitals.md

```diff
@@ -0,0 +1,174 @@
+
+Another option is to run Lighthouse in Chrome DevTools, described on [Google Developers](https://developers.google.com/web/tools/lighthouse#devtools) website.
+
+**See More**:
+- [Google's Guide on Optimizing LCP](https://web.dev/articles/optimize-lcp)
+
+## Interaction to Next Paint (INP)
+
+[Interaction to Next Paint (INP)](https://web.dev/articles/inp) measures how long a browser's paint cycle is blocked after a user interacts with a page. 
+
+Three areas where INP can be improved are:
+
+1. Input Delay - the time it takes for the browser to process user input and trigger event callbacks
+2. Event Callbacks - the time it takes for the browser to execute event callbacks after user input
+3. Rendering - the time it takes for the browser to render the page after user input
+
+We recommend reading Google's [Guide on Optimizing INP](https://web.dev/articles/inp) to learn more about how INP is calculated and some JavaScript techniques that can help improve it.
+
+For rendering, some common issues that can cause poor INP are:
+
+1. Complex CSS selectors - when a selector is too complex, the browser will have to parse and calculate the selector's specificity, which can slow down the rendering process after an interaction
+2. Large layout shifts - if an interaction causes a large layout shift, the browser will have to recalculate a large layout and then repaint the entire page, which can slow down the rendering process
+3. Large DOMs - the larger the DOM, the more expensive any rendering and updating operations will be
+
+
+**See More**:
+- [Google's Guide on Optimizing INP](https://web.dev/articles/inp)
+- [Finding Slow Interactions](https://web.dev/articles/optimize-inp#find_slow_interactions_in_the_field)
+- [Optimizing Long Tasks](https://web.dev/articles/optimize-long-tasks)
+
+## Cumulative Layout Shift (CLS)
+
+Cumulate Layout Shift (INP) is an important user-centric metric for measuring visual stability, which shows if page had any unexpected movement. It is a measure of stability in the layout of a page.
+
+Some examples of things that can increase layout shifts are images, web fonts, and dynamically loaded content.
+
+### Images
+
+When the browser parses the HTML, it will reserve the space for images based on their `width` and `height` attributes. If they are not defined - the browser will not reserve the extra space. So when the image is loaded, the image will cause the other elements on the page to shift around. 
+
+This can be avoided by setting the `width` and `height` attributes of the image. This way, even while the image is loading, the browser will account for it when it calculates the layout.
+
+```html
+<img
+  src="..."
+  width="128"
+  height="128"
+/>
+```
+
+### Web Fonts
```

Comment:
Definitely worth suggesting https://github.com/unjs/fontaine

Reply:
It's included at the end - good to know we're on the same page.